src/node_jsvmapi_types.h

```diff
@@ -21,7 +21,11 @@ enum napi_property_attributes {
   napi_default = 0,
   napi_read_only = 1 << 0,
   napi_dont_enum = 1 << 1,
-  napi_dont_delete = 1 << 2
+  napi_dont_delete = 1 << 2,
+
+  // Used with napi_define_class to distinguish static properties
+  // from instance properties. Ignored by napi_define_properties.
+  napi_static_property = 1 << 10,
```

Comment:
I wanted to leave space in case V8 adds new property attributes that we need to support. The lower attributes are literally the same constant values as the corresponding V8 attributes; they don't have to be but it makes conversion simpler.